src/sbpir/query.rs

```diff
@@ -211,4 +211,243 @@ mod tests {
         let expr5: Expr<Fr, Queriable<Fr>> = Expr::Pow(Box::new(Expr::Const(a)), 2);
         assert_eq!(format!("{:?}", expr5), "(0xa)^2");
     }
+
+    #[test]
+    fn test_add_fmt() {
+        let a: Fr = 10.into();
```

Comment:
@rutefig this test does not test something specific about this file?

Reply:
To test the operator you need to test with lhs and another with with rhs as queriables, otherwise you are using the expression operator

Reply:
You're right, removed as it didn't make sense